gslab_scons/_release_tools.py

```diff
@@ -10,29 +10,30 @@
 
 from _exception_classes import ReleaseError
 
-
-def release(vers, org, repo, 
+def release(vers, org, repo,
             DriveReleaseFiles = [],  
             local_release     = '',  
             target_commitish  = '', 
-            zip_release       = True):
+            zip_release       = True,
+            token             = None):
     '''Publish a release
 
     Parameters
     ----------
     env: an SCons environment object
     vers: the version of the release
     DriveReleaseFiles a optional list of files to be included in a
-        release to Google Drive.
+        release to drive (e.g. DropBox or Google Drive).
     local_release: The path of the release directory on the user's computer.
     org: The GtHub organisaton hosting the repository specified by `repo`.
     repo: The name of the GitHub repository from which the user is making
         the release.
     '''
     # Check the argument types
 
-
-    token    = getpass.getpass("Enter a GitHub token and then press enter: ") 
+    if token is None:
```

Comment:
I'm not sure this is best practice for passwords/tokens (i.e., having a user type it/enter on command line without hiding it).